lib/commands/context.js

```diff
@@ -353,7 +356,18 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
     bundleId: opts.chromeBundleId,
     useSystemExecutable: opts.chromedriverUseSystemExecutable,
     disableBuildCheck: opts.chromedriverDisableBuildCheck,
-  });
+  };
+
+  try {
+    this.ensureFeatureEnabled(CHROMEDRIVER_AUTODOWNLOAD_FEATURE);
```

Comment:
Could we just turn this into a helper method. Like

```javascript
isChromedriverAutoDownloadEnabled() {
try {
  this.ensureFeatureEnabled(CHROMEDRIVER_AUTODOWNLOAD_FEATURE);
  return true;
} catch (e) {
  log.debug(`Automated Chromedriver download is disabled. ` +
    `Use '${CHROMEDRIVER_AUTODOWNLOAD_FEATURE}' server feature to enable it`);
  return false;
}
}

Reply:
Then we can keep the old style the way it is and create the `Chromedriver` like the way it was before

```javascript
const chromedriver = new Chromedriver({	  const chromedriverOpts = {
    port: opts.chromedriverPort,	    port: opts.chromedriverPort,
    executable: opts.chromedriverExecutable,	    executable: opts.chromedriverExecutable,
    adb,	    adb,
    cmdArgs: opts.chromedriverArgs,	    cmdArgs: opts.chromedriverArgs,
    verbose: !!opts.showChromedriverLog,	    verbose: !!opts.showChromedriverLog,
    executableDir: opts.chromedriverExecutableDir,	    executableDir: opts.chromedriverExecutableDir,
    mappingPath: opts.chromedriverChromeMappingFile,	    mappingPath: opts.chromedriverChromeMappingFile,
    bundleId: opts.chromeBundleId,	    bundleId: opts.chromeBundleId,
    useSystemExecutable: opts.chromedriverUseSystemExecutable,	    useSystemExecutable: opts.chromedriverUseSystemExecutable,
    disableBuildCheck: opts.chromedriverDisableBuildCheck,	    disableBuildCheck: opts.chromedriverDisableBuildCheck,
    isAutodownloadEnabled : this. isChromedriverAutoDownloadEnabled()
  });
```

Reply:
yes, easy